src/backend/web/validation.js

```diff
@@ -102,9 +102,14 @@ const postsIdParamValidationRules = [
   param('id', 'Id Length is invalid').isLength({ min: 10, max: 10 }),
 ];
 
+const queryFeedsValidationRules = [
+  param('id', 'Id Length is invalid').isLength({ min: 10, max: 10 }),
```

Comment:
change this to feedsIdParamValidationRules to avoid any confusion with queryValidationRules from above

Reply:
I modified my pr again, I integrated checking the Param ID because Posts and Feeds check the length.
let me know if I need to improve my code more
Thank you :)

Reply:
@humphd : Should we have separate rules despite they are the same in length?